### PR TITLE
Ana/3929 add algo to storedwallet

### PR DESCRIPTION
**Pull request by Bitcoinera**
Necessary for #luniehq/lunie#4481

### CODE REVIEW

**Comment by faboweb** [comment]
Please change and store the following variables
```
curve = [sr25519, ed25519]
derivationPath
```

File: src/cosmos-keystore.ts [hunk 1]
```diff
@@ -38,15 +38,16 @@ export function storeWallet(
   name: string,
   password: string,
   network: string,
-  accountType: string,
+  HDPath: string,
+  curve: string,
```
Comment (faboweb) [inline src/cosmos-keystore.ts]: set a default here? this library only supports ed25519

File: src/cosmos-keystore.ts [hunk 2]
```diff
@@ -38,15 +38,16 @@ export function storeWallet(
   name: string,
   password: string,
   network: string,
-  accountType: string,
+  HDPath: string,
```
Comment (faboweb) [inline src/cosmos-keystore.ts]: set a default here? most Cosmos networks use the Cosmos HD path

File: src/cosmos-keystore.ts (hunk 1 above)
Comment (Bitcoinera) [reply src/cosmos-keystore.ts]: oh, ok

**Review by faboweb — APPROVED** [review]

**Pull request merged**